### PR TITLE
List: prevent falling through checkbox in firefox

### LINKED ISSUE #605: When enabling/disabling name autoextend the name screen is shown
Browser: Firefox
Version:  Superhero ff63864 / v.0.4.0 

When clicking the auto-extend tickbox on a name which I own I'm sent to the screen with name details. When I click the tickbox I'm not expecting to be sent to another screen. Going back to my names it turns out the tickbox was indeed activated.

### PR DESCRIPTION
**Pull request by etharner**
Closes #605.

### CODE REVIEW

File: src/popup/router/pages/Names/List.vue [hunk 1]
```diff
@@ -22,7 +22,8 @@
         <span class="address">{{ owner }}</span>
         <CheckBox
           :value="autoExtend"
-          @input="value => $store.commit('names/setAutoExtend', { index, value })"
+          prevent
+          @click.native="$store.commit('names/setAutoExtend', { index, value: !autoExtend })"
```
Comment (davidyuk) [inline src/popup/router/pages/Names/List.vue:26]:
```suggestion
          @input="value => $store.commit('names/setAutoExtend', { index, value })"
          @click.native.prevent
```

I think CheckBox's `prevent` shouldn't be used, can't it work this way? To use input event with CheckBoxes looks more semantic.

Comment (davidyuk) [reply src/popup/router/pages/Names/List.vue:26]: And let's don't put clickable inside clickable the next time...

Comment (etharner) [reply src/popup/router/pages/Names/List.vue:26]: I have this issue in firefox both in extension and web-wallet and I've tried this solution but it doesn't work 🤷

**Comment by Liubov-crypto** [comment]
I can't reproduce this error. When I click on Auto Extend in my wallet it doesn't redirect me anywhere

**Comment by mradkov** [comment]
> I can't reproduce this error. When I click on Auto Extend in my wallet it doesn't redirect me anywhere

Same here.

**Review by davidyuk — APPROVED** [review]
I can't reproduce this on macOS, but can in VM with Ubuntu😄 let's merge it as it is, and refactor CheckBox component and it's usages later.

**Pull request merged**